config/environments/development.rb

```diff
@@ -33,11 +42,11 @@
   config.active_storage.service = :local
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
 
   config.action_mailer.perform_caching = false
-
-  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
+  #
+  #config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
Why does this get commented out?

Reply:
I set it to the mg.rideboard.app instead